test/Misc/serialized-diagnostics-batch-mode-nonprimary-suppression.swift

```diff
@@ -3,24 +3,15 @@
 //
 // RUN: rm -f %t.*
 
-// RUN: not %target-swift-frontend -typecheck  -primary-file %s  -serialize-diagnostics-path %t.main.dia -primary-file %S/../Inputs/empty.swift  -serialize-diagnostics-path %t.empty.dia %S/Inputs/serialized-diagnostics-batch-mode-suppression-helper.swift    2> %t.stderr.txt
+// RUN: not %target-swift-frontend -typecheck  -primary-file %s  -serialize-diagnostics-path %t.main.dia -primary-file %S/../Inputs/empty.swift  -serialize-diagnostics-path %t.empty.dia %S/Inputs/serialized-diagnostics-batch-mode-suppression-helper.swift  2> %t.stderr.txt
 // RUN: c-index-test -read-diagnostics %t.main.dia 2> %t.main.txt
 // RUN: c-index-test -read-diagnostics %t.empty.dia 2> %t.empty.txt
 
-// Ensure there was an error:
+// RUN: %FileCheck -check-prefix=NO-COMPILATION-FAILED %s <%t.main.txt
+// RUN: %FileCheck -check-prefix=NO-COMPILATION-FAILED %s <%t.empty.txt
+// NO-COMPILATION-FAILED-NOT: compilation failed
```

Comment:
You changed the text of the error, so this isn't up to date anymore. (A hazard of negative checks!)

Reply:
Thanks! Fixed.